### PR TITLE
Pick up kube credentials from env variables for k8s integration test

### LINKED ISSUE #276: Fix Kubernetes test
The kubernetes test requires a kube config that needs to be created and added to the test. In addition, this requires the `KubernetesContainerManager` constructor to take an optional config argument.

### PR DESCRIPTION
**Pull request by dcrankshaw**
Fixes #276 

This should be the last PR needed for the 0.2 release.

Have a release taco 🌮  🌮  🌮 !!

### CODE REVIEW

**Comment by dcrankshaw** [comment]
jenkins test this please

File: bin/ci_checks.sh [hunk 1]
```diff
@@ -16,8 +23,25 @@ tag=$(<VERSION.txt)
 cd -
 
 # Test docker login
-docker pull 568959175238.dkr.ecr.us-west-1.amazonaws.com/clipper/query_frontend:$tag
+# docker pull 568959175238.dkr.ecr.us-west-1.amazonaws.com/clipper/query_frontend:$tag
```
Comment (Corey-Zumar) [inline bin/ci_checks.sh]: Should this be removed or uncommented?

**Review by Corey-Zumar — APPROVED** [review]
LGTM. Release taco inbound! :fireworks: :taco: :tada: :taco:

File: bin/ci_checks.sh (hunk 1 above)
Comment (dcrankshaw) [reply bin/ci_checks.sh]: Uncommented

**Pull request merged**